README.md

```diff
@@ -61,6 +61,7 @@ Options:
   -t|--target             Path to the test runner application.
   -a|--targetargs         Arguments to be passed to the test runner.
   -o|--output             Output of the generated coverage report
+  -v|--verbosity          Sets the verbosity level of the command. Allowed values are quiet, minimal, normal, detailed.
```

Comment:
I did some test with coverlet tool and msbuild and this PR...and seem that is not possible keep same behaviour between msbuild logger and console logger levels.
I mean we want behave in a different way if we use command or msbuild(msbuild rule are not under our control), and to do this we need to add some "flag" on interface(i.e. `important` that is not honored on msbuild, not so clean).
Again maybe if we support verbosity we should concat it as a `--targetargs`?

I wonder if could be more "clear" avoid levels at all(my first idea with better hope of success :smile:), and simply add `-quiet` switch, and nop the console.
The idea to see all coverlet log(on coloured console, red yellow white) is not so bad to me, we explicity run `coverlet` tool command so it's expected some logs from it.
But it's also true that someone would want not see anything and only verify "reports" at the end.

After Viktor's attempt(thank's a lot!) I don't like the idea to mix two different behaviour under same "verbosity concepts" that cannot be honorated except with some "non clean" solution. 
What do you think Viktor?

I would like to know @tonerdo's thought

Reply:
Why are different log outputs between msbuild and coverlet.console an issue for you? The former is running as part of an assembly's build, the latter is a standalone tool. I would not expect to have identical outputs.

Reply:
> Why are different log outputs between msbuild and coverlet.console an issue for you?

I would expect that output using msbuild or using coverlet tool should be the same for same level, for instance if I would like to pass `minimal` to tool and msbuild and get same output(at the moment we show the list of instrumented assemblies), and possibly avoid to change contract(log interface) only to accomodate one scenario.

Reply:
@tonerdo is [not a "fan"](https://github.com/tonerdo/coverlet/issues/388#issuecomment-488676943) of my sync idea...so I lose 😄 
I think we can go on with "unsync" one.